src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -1018,6 +1018,11 @@ def sdc_pin_constraints(self) -> str:
                 direction=delay.direction,
                 name=delay.name
             ))
+
+        # Custom pin constraints that are verbatim appended
```

Comment:
```suggestion
        # Custom sdc constraints that are verbatim appended
```
Also `verbatim appended` sounds off to me. `appended verbatim` sounds better but I don't particularly care.